### PR TITLE
Return user bucks after feature and bet

### LINKED ISSUE #199: Need to show feedback that submitting a feature request charges a user 250 BillyBucks

### PR DESCRIPTION
**Pull request by flogan**
Return new user bucks after feature request and challenge bet

resolves [issue #199](https://github.com/BillyP-Bot/BT-Bots/issues/199)

### CODE REVIEW

File: src/controllers/challenges/index.ts [hunk 1]
```diff
@@ -122,7 +122,7 @@ export const challengeRouter = async function (app: FastifyInstance) {
 					{ $inc: { billy_bucks: -amount, "metrics.gambling.challenges.bets": 1 } }
 				)
 			]);
-			return result;
+			return { bet, billyBucks: user.billy_bucks };
```
Comment (judegiordano) [inline src/controllers/challenges/index.ts]: this is fine but im pretty sure most of our other endpoint return snake_case

File: src/controllers/features/index.ts [hunk 2]
```diff
@@ -21,7 +21,7 @@ export const featureRouter = async function (app: FastifyInstance) {
 					}
 				},
 				response: {
-					200: { $ref: "feature#" }
+					200: { feature: { $ref: "feature#" }, billyBucks: { type: "number" } }
```
Comment (judegiordano) [inline src/controllers/features/index.ts]: meaning this would change as well

File: src/controllers/features/index.ts [hunk 3]
```diff
@@ -43,7 +43,7 @@ export const featureRouter = async function (app: FastifyInstance) {
 					}
 				)
 			]);
-			return result;
+			return { feature: featureResult, billyBucks: userResult.billy_bucks };
```
Comment (judegiordano) [inline src/controllers/features/index.ts]: same as above

**Review by judegiordano — CHANGES_REQUESTED** [review]

File: src/controllers/challenges/index.ts (hunk 1 above)
Comment (flogan) [reply src/controllers/challenges/index.ts]: ah yes, good catch. thanks

**Review by cdleveille — APPROVED** [review]

**Pull request merged**